README.md

```diff
@@ -0,0 +1,274 @@
+# It uses the "videos" storage where you can configure the storage backend you want to use.
+STORAGES = {
+    "default": {
+        "BACKEND": "django.core.files.storage.FileSystemStorage",
+    },
+    "videos": {  # This is the storage used to store the transcoded videos
+        "BACKEND": "app.storage.MyCustomFileSystemVideoStorage", # You can use the storage backend you want
+    },
+    "staticfiles": {
+        "BACKEND": "django.contrib.staticfiles.storage.StaticFilesStorage",
+    },
+}
+```
+
+
+#### Storage
+
+Django-peertube-runner-connector uses the django storage system to store the transcoded videos. It uses the "videos" storage where you can configure the storage backend you want to use. To use an S3 like storage, you can use the [django-storages](https://django-storages.readthedocs.io/en/latest/) library. Here is an example of a custom storage backend that uses the S3 storage:
+
+```python
+# app/storage.py
+from storages.backends.s3boto3 import S3Boto3Storage
+
+
+class MyS3VideoStorage(S3Boto3Storage):
+  """Custom S3 storage class."""
+
+  bucket_name = "my-bucket"
+```
+
+then you can use it in your settings:
+
+```python
+# settings.py
+
+# ... S3 settings
+
+STORAGES = {
+    "default": {
+        "BACKEND": "django.core.files.storage.FileSystemStorage",
+    },
+    "videos": {  
+        "BACKEND": "app.storage.MyS3VideoStorage", # Your custom storage backend
+    },
+    "staticfiles": {
+        "BACKEND": "django.contrib.staticfiles.storage.StaticFilesStorage",
+    },
+}
+```
+
+#### Server
+
+To make use of the SocketIO server, you need to have ASGi server like [uvicorn](https://www.uvicorn.org/).
+
+Here is an example on how to configure your asgi server to use the SocketIO server:
+
+
+```python
+from configurations.asgi import get_asgi_application
+
+django_asgi_app = get_asgi_application()
+
+
+# its important to make all other imports below this comment
+import socketio 
+
+from django_peertube_runner_connector.socket import sio 
+
+
+application = socketio.ASGIApp(sio, django_asgi_app)
+
+```
+
+Add the runners api views to your urls:
+
+```python
+# urls.py
+from django_peertube_runner_connector.urls import (
+    urlpatterns as django_peertube_runner_connector_urls,
+)
+
+urlpatterns += django_peertube_runner_connector_urls
+```
+
+Voilà! Your server should be ready!
+
+### Demo application
+
+For testing purpose, you can find a basic django app using the django-peertube-runner-connector library in the `tests` directory. You use run it with the following commands:
+
+Create your virtual environment:
+```shell
+python -m venv env
+source env/bin/activate
+```
+
+Install the dependencies:
+```shell
+pip install -e ."[dev]"
+```
+
+Go to the tests directory:
+```shell
+cd tests
+```
+
+Create the database and run the migrations:
+```shell
+python manage.py migrate
+```
+
+
+Launch the server with an asgi server like uvicorn:
+```shell
+python -m uvicorn app.asgi:application --reload
+```
```

Comment:
Admin should be accessible from [http://127.0.0.1:8000/admin/](http://127.0.0.1:8000/admin/)

Reply:
ditto @polyhb